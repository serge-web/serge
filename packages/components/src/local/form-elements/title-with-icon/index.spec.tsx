/* global it expect */

import React from 'react'
import renderer from 'react-test-renderer'

import TitleWithIcon from './index'

it('TitleWithIcon renders correctly', () => {
  const tree = renderer
    .create(<TitleWithIcon forceColor="blue" icon="merchant-vessel.svg">Test</TitleWithIcon>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
