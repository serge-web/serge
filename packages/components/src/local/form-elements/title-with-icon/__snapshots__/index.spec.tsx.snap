// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleWithIcon renders correctly 1`] = `
<div
  className="main"
>
  <h2>
    Test
  </h2>
  <div
    className="icon"
  >
    <div
      className="asset-icon-background"
      style={
        Object {
          "backgroundColor": "blue",
        }
      }
    >
      <div
        className="asset-icon-with-image"
      >
        <img
          alt="merchant-vessel"
          className="img"
          src="/assets/counters/merchant-vessel.svg"
        />
      </div>
    </div>
  </div>
</div>
`;
